test/stream/api/api_test.go

```diff
@@ -4,42 +4,34 @@ package api
 import (
 	"context"
 	"runtime"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	etcd "go.etcd.io/etcd/client/v3"
+	"github.com/stretchr/testify/require"
 
-	"github.com/keboola/keboola-as-code/internal/pkg/encoding/json"
 	"github.com/keboola/keboola-as-code/internal/pkg/env"
 	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/etcdhelper"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper/runner"
 )
 
-const (
-	receiverSecretPlaceholder = "<<RECEIVER_SECRET>>"
-)
-
-// TestBufferApiE2E runs one Buffer API functional test per each subdirectory.
-func TestBufferApiE2E(t *testing.T) {
+// TestStreamApiE2E runs one Stream API functional test per each subdirectory.
+func TestStreamApiE2E(t *testing.T) {
 	t.Parallel()
```

Comment:
I re-enabled Steram API E2E tests, with `api-startup` and `endpoint-not-found` basic tests.